migrate.py

```diff
@@ -468,12 +472,11 @@ def format_comment_body(comment, options):
     data = dict(
         author=format_user(author, options),
         sep='-' * 40,
+        content=content,
     )
-    if author and author['username'] == options.bb_skip:
-        header = ""
-    else:
-        header = COMMENT_HEADER.format(**data)
-    return header + content
+    skip_user = author and author['username'] == options.bb_skip
```

Comment:
Personally, I find this style a little difficult to read because it looks like it's assigning `skip_user = author` only to be jarred by the `and ...` clause...

So I prefer to surround the compound boolean check with parens `z = (w and x == y)` for improved readability. Not a blocker to merge, just a thought...

Reply:
Readability: matter of taste I guess. Not sure whether some pep8/flake8/whatever checker would complain about superfluous parentheses then.

Just think of it as computing a boolean on the rhs, assigning to the lhs.
Same boolean as the previous code use in the if-condition, btw.